### PR TITLE
Create Request For Web Part 1

### PR DESCRIPTION
**Pull request by sondth-1037**
## Related Tickets:
- [#27888](https://edu-redmine.sun-asterisk.vn/issues/27888)

## WHAT this PR does?
- Create Request For Anomynous User Part 1

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by sondth-1037** [comment]
ready

File: resources/lang/en/contents.php [hunk 1]
```diff
@@ -12,6 +12,11 @@
             'name' => 'Your name',
             'phone' => 'Phone number',
```
Comment (huyenttm-0901) [inline resources/lang/en/contents.php]: viết cả file vi đi em

Comment (sondth-1037) [reply resources/lang/en/contents.php]: vâng ạ

**Comment by sondth-1037** [comment]
ready

**Pull request merged**